packages/DataGrid/src/components/Cell/Cell.js

```diff
@@ -41,18 +48,47 @@ export default function Cell(props) {
 
   const cellProps = typeof column.cellProps === "function" ? column.cellProps(options) : {};
 
+  function setHighlight(value = false) {
+    setIsHighlighted(value);
+  }
+
+  React.useEffect(() => {
+    const key = getGridRefId({ gridId, rowIndex, columnIndex });
+    window.paprika.dataGridRef[key] = {
+      setHighlight,
+    };
+
+    return () => {
+      console.log("before:", Object.keys(window.paprika.dataGridRef).length);
+      delete window.paprika.dataGridRef[key];
+      console.log("after:", Object.keys(window.paprika.dataGridRef).length);
+    };
+  }, []); // eslint-disable-line
+
+  const isColumnCellAFunction = typeof column.cell === "function";
   return (
     <sc.Cell
       borderType={borderType}
-      data-cell={`${gridId}.${columnIndex}.${rowIndex}`}
+      data-pka-cell-key-column-index={columnIndex}
+      data-pka-cell-key-row-index={rowIndex}
+      data-pka-cell-key={`${gridId}.${columnIndex}.${rowIndex}`}
+      data-pka="cell-container"
       hasZebraStripes={hasZebraStripes}
       rowIndex={rowIndex}
       style={style}
       tabIndex={-1}
     >
-      <sc.GridCell role="gridcell">{a11yText}</sc.GridCell>
-      <sc.InnerCell {...cellProps} aria-hidden="true" {...options.attrs}>
-        {typeof column.cell === "function" ? column.cell(options) : data[rowIndex][column.cell]}
+      <sc.GridCell data-pka="cell-a11y" role="gridcell">
```

Comment:
Recommend `data-pka-anchor='data-grid.gridcell'` (or `grid-cell`) to align with the aria role, rather than just `a11y`.